documentation/src/main/asciidoc/upgrading/upgrading-content.adoc

```diff
@@ -1,5 +1,13 @@
 == Upgrading from 9.4 to 10.0
 
+=== Removed AtomicMap and FineGrainedAtomicMap
+
+AtomicMapLookup, AtomicMap and FineGrainedAtomicMap have been removed. No direct replacement exists.
```

Comment:
We can probably point them to use the functional methods (CHM.merge or FunctionalMap) to do similar operations instead.